service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FeedbackReceiver.java

```diff
@@ -39,9 +39,9 @@
      * @param iotHubServiceClientProtocol protocol to be used
      *
      */
-    public FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol)
+    FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol)
```

Comment:
Add the private keyword?

Reply:
This constructor is called from another class within this package, so I can't mark it as private. This lack of modifier makes the constructor "Package private" which means only classes within this package can access it. Classes outside of the package, even if they extend this class, can't access it